package/src/RNCSliderNativeComponent.web.tsx

```diff
@@ -318,8 +324,10 @@ const RCTSliderWebComponent = React.forwardRef(
       <View
         ref={containerRef}
         onLayout={({nativeEvent: {layout}}: LayoutChangeEvent) => {
+          const lowerValue = lowerLimit === minimumValue ? 0 : lowerLimit;
```

Comment:
I'm not sure if I understand this implementation - we put `0` if `lowerLimit` is equal to `minimumValue`?
Why? What if user set's `lowerLimit={10}` and `minimumValue={10}`? We would still be putting 0 here?